onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -106,6 +106,14 @@ struct AsyncResource {
     fetch_names.reserve(sz);
     fetch_names_raw.reserve(sz);
   }
+
+  ~AsyncResource() {
+    for (auto* fetch : fetches_raw) {
```

Comment:
```suggestion
    for (const auto* fetch : fetches_raw) {
```

Reply:
with const *, we wouldn't be able to utilize a unique_ptr to recycle the fetch.
An alternative is simply do "delete fetch", while last time we got warnings from pipeline for calling "delete" directly.

Reply:
Done.